docs/bugs.json

```diff
@@ -1,14 +1,22 @@
 [
     {
+        "name": "DelegateCallReturnValue",
+        "summary": "The low-level .delegatecall() took the returned value as a boolean instead of the execution outcome.",
+        "description": "If the destination would return a value, which interpreted as a boolean means false, the return value of .delegatecall() would be false, despite successful execution.",
```

Comment:
Ah, then I think we should write that without a comma. But what about writing "If the called contract returns zero, ..."?